server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/http/SessionSupplier.scala

```diff
@@ -0,0 +1,64 @@
+/** **************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ * *
+ * http://www.apache.org/licenses/LICENSE-2.0                 *
+ * *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ * ***************************************************************/
+
+package org.apache.james.jmap.http
+
+import org.apache.james.core.Username
+import org.apache.james.jmap.model._
+
+object SessionSupplier {
+  private val CORE_CAPABILITY = CoreCapability(
+    UnsignedInt(10_000_000), UnsignedInt(4), UnsignedInt(10_000_000),
```

Comment:
please use named arguments

Reply:
I had received opposite comments on this kind of coding convention. Some time I was asked to remove named parameters, sometime, value are better than passing constants. Could we have an unified convention? when to use named parameter, when not? when to use constants?

Reply:
```
should use named parameter and avoid constants when possible
```

is that okay?

Reply:
You should do:

```
CoreCapability(
   maxRequestSize = 10_000_000,
   ....
```

That way every one know what 10_000_000 means and we don't need a constant or a builder to give it a name

Reply:
or even  better you strong type all the UnsignedInt